src/__tests__/user.spec.js

```diff
@@ -1,18 +1,73 @@
 const request = require("supertest");
 const app = require("../../index");
 const db = require("../db/testDb");
+const { createSignature } = require("@leapchain/dleap");
+const getOrCreateUser = require("../utils/getOrCreateUser");
+const {
+  GOVERNOR_REQUEST_FEE,
+  TREASURY_ACCOUNT_NUMBER,
+  MemoType,
+} = require("../constants");
 
-beforeAll(async () => await db.connect());
+beforeAll(async () => {
+  await db.connect();
+  secretKey =
+    "705a3b36b44c2e95eebba870dc6fe5280c5c72783e2fe9449ebb73556faec57e";
+  publicKey =
+    "a5f0766103d33981ca0e90179fbd7ece2c9e76e0ea5d99ebb86cb80ff3c99ab9";
+});
 beforeEach(async () => await db.clear());
 afterAll(async () => await db.close());
 
+const generateUserJWT = async () => {
+  const user = await getOrCreateUser(publicKey);
+  const message = {
+    nonce: user.nonce,
+  };
+  const signature = createSignature(JSON.stringify(message), secretKey);
+
+  const res = await request(app)
+    .post("/api/v1/auth")
+    .send({
+      accountNumber: publicKey,
+      message: {
+        nonce: user.nonce,
+      },
+      signature: signature,
+    })
+    .set("Accept", "application/json");
+  return { accessToken: res.body.accessToken, userId: user._id };
+};
+
+const generateGovernorJWT = async () => {
+  const user = await getOrCreateUser(publicKey);
+  user.type = "GOVERNER";
+  user.save();
+  const message = {
+    nonce: user.nonce,
+  };
+  const signature = createSignature(JSON.stringify(message), secretKey);
+
+  const res = await request(app)
+    .post("/api/v1/auth")
+    .send({
+      accountNumber: publicKey,
+      message: {
+        nonce: user.nonce,
+      },
+      signature: signature,
+    })
+    .set("Accept", "application/json");
+  return { accessToken: res.body.accessToken };
+};
+
 describe("POST /api/v1/users/create", () => {
   it("should return account number and nonce for valid request", async () => {
     const res = await request(app)
       .post("/api/v1/users/create")
       .send({
         accountNumber:
-          "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa",
+          "a5f0766103d33981ca0e90179fbd7ece2c9e76e0ea5d99ebb86cb80ff3c99ab9",
       })
```

Comment:
you should use env to run a test-acc creation

Reply:
changed the credentials to account pair for tests